org.lflang.tests/src/org/lflang/tests/TestBase.java

```diff
@@ -606,7 +607,7 @@ private List<ProcessBuilder> getFederatedDockerExecCommand(LFTest test) {
             System.out.println(Message.MISSING_DOCKER);
             return List.of(new ProcessBuilder("exit", "1"));
         }
-        var srcGenPath = test.context.getFileConfig().getSrcGenPath();
+        var srcGenPath = new FedFileConfig.FedFiles(test.context.getFileConfig()).getFedSrcGenPath();
```

Comment:
I think the right fix is to remove `getFedSrcGenPath` and put its contents in an override of `getSrcGenPath`. WDYT?

Reply:
Yes, the way it is done now is a little weird. I am certainly open to refactoring, but I just wanted to first get to something that works.

Reply:
I've fixed this because I had to for another reason in 1cc48459e. I hope it still works.